GVRf/Framework/framework/src/main/java/org/gearvrf/script/GVRScriptManager.java

```diff
@@ -217,22 +250,94 @@ public GVRScriptFile getScriptFile(IScriptable target) {
      * @throws GVRScriptException if script processing error occurs.
      */
     public GVRScriptFile loadScript(GVRAndroidResource resource, String language) throws IOException, GVRScriptException {
-        if (getEngine(language) == null) {
-            mGvrContext.logError("Script language " + language + " unsupported", this);
-            throw new GVRScriptException(String.format("The language is unknown: %s", language));
-        }
+        //{
```

Comment:
This method could use a cleanup

Reply:
Yes, this code is still a work-in-progress and the Pull Request was only done to allow Tom to test his application.

Reply:
Thanks for clarifying. Updated the status so everybody knows.